exonum-java-binding-core/rust/integration_tests/tests/service_proxy.rs

```diff
@@ -105,11 +89,12 @@ fn tx_from_raw() {
         *INFO_VALUE
     );
 }
+*/
 
 #[test]
 #[should_panic(expected = "Java exception: java.lang.OutOfMemoryError")]
 fn tx_from_raw_should_panic_if_java_error_occurred() {
-    let raw = RawTransaction::from_vec(vec![]);
+    let raw = RawTransaction::new(0, ServiceTransaction::from_raw_unchecked(0, vec![]));
```

Comment:
I'd extract this repeated code in a method or a builder with sensible defaults, so that we can ignore unrelated id values.

Reply:
Done, thanks for spotting